src/Engines/Nunjucks.js

```diff
@@ -4,6 +4,78 @@ const TemplatePath = require("../TemplatePath");
 const EleventyErrorUtil = require("../EleventyErrorUtil");
 const EleventyBaseError = require("../EleventyBaseError");
 
+/*
+ * HACKITYHACKHACKHACKHACK
+ */
+let inc = ((i) => {
+  return () => {
+    return i++;
+  };
+})(0);
+let pathMap = new Map();
+
+let clearCache = (reason = "") => {
```

Comment:
The `reason` argument gets passed in, but seems to be unused within this function.

Reply:
Removed in updated patch.